src/scripts/OSFramework/OSUI/Pattern/Tabs/scss/_tabs.scss

```diff
@@ -234,6 +234,10 @@
 				-servicestudio-margin-bottom: var(--space-m);
 			}
 
+			& div:empty {
```

Comment:
This should be inside the selector on line 232

Reply:
Fixed